src/Paprika.Runner/StatisticsForPagedDb.cs

```diff
@@ -77,12 +77,11 @@ private static Layout BuildReport(StatisticsVisitor.Stats stats, string name, in
 
         var t = new Table();
         t.AddColumn(new TableColumn("Depth"));
-        t.AddColumn(new TableColumn("Page count"));
-        t.AddColumn(new TableColumn("Leaf page count"));
-        t.AddColumn(new TableColumn("Overflow page count"));
-        t.AddColumn(new TableColumn($"{nameof(DataPage)}-inner (P50) % usage"));
-        t.AddColumn(new TableColumn($"{nameof(DataPage)}-leaf (P50) % usage"));
-        t.AddColumn(new TableColumn($"{nameof(LeafOverflowPage)} (P50) % usage"));
+        t.AddColumn(new TableColumn("Total page count"));
+        t.AddColumn(new TableColumn("Data page count"));
+        t.AddColumn(new TableColumn("Bottom page count"));
+        t.AddColumn(new TableColumn($"{nameof(DataPage)} P50 % usage"));
```

Comment:
I'm not sure I'm following. This is a static view that you can run with `Cli stats` (added it to the description as additional picture). If we're talking about metrics we do aggregate some of them with `System.Diagnostics.Metrics`. We don't gather percentage used etc as it would be hard (update every time?). I think I'm not following the comment fully. Could you rephrase?

Reply:
Hmm, got it - this is just the static view once the node is fully synced. What I meant was that could this static view be captured across time intervals (say every few mins?). I agree that it would be compute intensive, but would that data be useful to understand the trend/correlation at different times?